phalcon/mailer.zep

```diff
@@ -0,0 +1,624 @@
+    *     'server'     => "server.server.it",   // sets the SMTP server
+    *     'port'       => 587,                  // set the SMTP port for the GMAIL server
+    *     'username'   => "noreply@server.it",  // SMTP account username
+    *     'password'   => "password",           // SMTP account password
+    *     'secure'     => "tls",
+    *     ];
+    *
+    * // Create the object Mailer
+    * // set in mail the value of the config
+    * $objMail = new Mailer($config);
+    *
+    * // Set address and name for:
+    * // -From where
+    * // -Answer to
+    * $objMail->setFrom('prueba_from@address.com', 'pruba_name_from');
+    * $objMail->replyTo('prueba_reply_to@address.com', 'pruba_name_reply_to');
+    *
+    * // Insert the mail content, message and attached information
+    * $objMail->msg('mensaje de prueba');
+    * $objMail->addAttachment('/file.odt');
+    *
+    * // Add addressee for to
+    * $objMail->addTo('prueba_to@address.com', 'pruba_name_to');
+    *
+    * // Send mail
+    * $objMail->send();
+    * </code>
+    *
+    **/
+
+class Mailer {
+    protected server {
+        set, get
+    };
+    protected port {
+        set, get
+    };
+    protected username {
+        set, get
+    };
+    protected password {
+        set, get
+    };
+    protected secure{
+        set, get
+    };
+    
+    protected smtpAuth = false {
+        set, get
+    };
+    protected xMailer = "Phalcon Mailer" {
+        set, get
+    };
+    protected priority = 3 {
+        set, get
+    };
+    
+    private from = "";
+    private fromName = "";
+    private replyTo = "";
+    private replyToName = "";
+    private recipients = [];
+    
+    protected subject = "" {
+        set, get
+    };
+    protected message = "" {
+        set, get
+    };
+    
+    protected charset = "\"utf-8\"" { 
+        set, get
+    };
+    
+    /**
+     * Gett and Sett contentType
+     * Can be set to: text/plain, text/html, multipart/mixed
+     *
+     * @param string 
+     * 
+     */
+    protected contentType = "multipart/alternative" { 
+        set, get
+    };
+    
+    protected transferEncodeing = "8bit" { 
+        set, get
+    }; 
+    protected altBody = "" { 
+        set, get
+    };
+    protected isLogin = false { 
+        set, get
+    };
+    protected cc = []{ 
+        set, get
+    };
+    protected bcc = [] { 
+        set, get
+    };
+    protected attachments = [] { 
+        set, get
+    };
+
+    private conn;
+    private newline = "\r\n";
+    private localhost = "localhost";
+    private timeout = "15";
+    private debug = false;
+    
+    /**
+     * Constructor
+     */
+    public function __construct(var parameters){
+        this->setParams(parameters);
+    }
+    
+    /**
+     * Set parameters
+     *
+     * @param array   parameters
+     * 
+     */
+    public function setParams(var parameters) -> void {
+        var key, value;
+        
+        if (typeof parameters != "array") {
```

Comment:
Instead of this if and exception just change `var parameters` to `array parameters`.